app/components/developer_query_component.html.erb

```diff
@@ -83,8 +83,88 @@
               </div>
             </div>
           </div>
+    <% end %>
+    <div data-controller="toggle" data-toggle-close-class="hidden" class="relative z-10 inline-block text-left sm:hidden">
+      <div>
+        <button type="button" data-action="toggle#toggle" class="group inline-flex items-center justify-center text-sm font-medium text-gray-700 hover:text-gray-900" aria-expanded="false">
```

Comment:
I don't think we need the following CSS classes: `group inline-flex items-center justify-center`.